presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchConnectorConfig.java

```diff
@@ -35,6 +35,15 @@
     private Duration requestTimeout = new Duration(100, MILLISECONDS);
     private int maxRequestRetries = 5;
     private Duration maxRetryTime = new Duration(10, SECONDS);
+    private String certificateFormat = "";
```

Comment:
shall we define an enum for the format? using PEM, JKS, and NONE as enum values, so that we could get rid of upper case string comparison